lib/StackAnalysis/StackAnalysis.cpp

```diff
@@ -30,14 +61,22 @@
 using llvm::BasicBlock;
 using llvm::Function;
 using llvm::Module;
+using llvm::raw_fd_ostream;
 using llvm::RegisterPass;
 
-static Logger<> ClobberedLog("clobbered");
-static Logger<> StackAnalysisLog("stackanalysis");
-static Logger<> CFEPLog("cfep");
+using FunctionEdgeTypeValue = model::FunctionEdgeType::Values;
+using FunctionTypeValue = model::FunctionType::Values;
+using GCBI = GeneratedCodeBasicInfo;
+
+using GlobalsArrayRef = llvm::ArrayRef<llvm::GlobalVariable *>;
+using GlobalsSet = std::set<llvm::GlobalVariable *>;
```

Comment:
Drop or move into classes that need these.

Reply:
Dropped.